custom/shared/contexts/CallProvider.js

```diff
@@ -52,9 +52,10 @@ export const CallProvider = ({
     if (!daily) return;
     const updateRoomConfigState = async () => {
       const roomConfig = await daily.room();
-      if (!('config' in roomConfig)) return;
+      const config = roomConfig?.config;
+      if (!config) return;
 
-      if (roomConfig?.config?.exp) {
+      if (config.exp) {
         setRoomExp(
           roomConfig?.config?.exp * 1000 || Date.now() + 1 * 60 * 1000
```

Comment:
Could also reuse the `config` var here vs retrieving it again I think, since we know it's available!

Reply:
@lizadaily , great catch, thank you so much! I just updated `setRoomExp` to use the variable. Let me know if that looks better to you!

Reply:
Nice! It does look better this way imo, but also I suspect limiting use of optional chaining will usually result in quicker execution. Did a quick benchmark test here: https://www.measurethat.net/Benchmarks/Show/16207/0/optional-chaining-vs-reuse
(Maybe for the purpose of an example it isn’t really a big impact, but seems like we may as well avoid  unnecessary overheard where possible)